docs/dev/IDL.md

```diff
@@ -30,6 +32,17 @@ declaration, similar to C++.
 Some places (e.g. classes, class members and RPC verbs) also support modifying generation behavior by providing
 attribute sequences (via C++ syntax of `[[attr]]` form).
 
+### Include and import
+
+Data types that are defined elsewhere can be used in IDL definition files.
+
+To include definitions from C++ headers, use the `include "filename.hh"` statement.
+The idl compiler generates a respective `#include "filename.hh"` directive in the generated `<mod_name>.dist.hh` file.
+
+To import definitions from other IDL definition files, use the `import <other_mod_name>` statement.
+The idl compiler generates a respective `#include "<other_mod_name>.dist.hh"` directive in the generated `<mod_name>.dist.hh` file
+and a `#include "<other_mod_name>.dist.impl.hh"` directive in the generated `<mod_name>.dist.impl.hh` file
+
```

Comment:
IDL was designed to mimic C++ syntax, so we can switch to C++ reflection when it becomes available (optimistically in C++26). So I'd rather not add new syntax, which conflicts with C++ module import.

Can we make #include detect that it's including idl? For example, #include "xyz.idl.hh" is importing an idl module.

Reply:
I don't know how

Reply:
str.endswith(".idl.hh")?

Reply:
ah, so it will look like C pre-processor, but will do more.
that's possible.